setup.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
```

Comment:
I dont think a full addition to the README is necessary ultimately maybe just a one line comment here saying 
`#run this file if you are unsure of your installed node-version`

Reply:
```suggestion

#Run this file if you are unsure of your installed node version
```